pages/playerLayout.js

```diff
@@ -0,0 +1,100 @@
+import { Box, Card, Image, Heading, Text, Flex, Button } from "rebass";
```

Comment:
Maybe we can stash this work then branch off master and work on it from there? Best to keep different aspects seperate IMO.

Reply:
Sounds good to me

Reply:
In that case, /pages/api/upload.js should be stashed too

Reply:
Yep and also some changes to index.js removed